### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by kgrimley-bu** [bmlt-enabled/wordpress-bmlt-workflow#17]
Only a couple that I'm seeing.

1) color contrast - 
    - the "x" for removing meeting formats is 2.24 and minimum contrast should be 4.5 Darkening the "x" to match the text will solve this one
    - The color/background color combo for uneditable fields on the Close meeting form are 2.44. Should lighten one to meet minimum contrast

2) Missing form labels
    - Online meeting link
    - Additional Information
Both need the for attribute on the label element

3) Duplicate form labels - Both the "Reason for Update" and "Search for Meeting" labels have the for attribute as "update_reason"

There's an additional potential missing form label with select2. It could be a false positive as the dev is pretty active with accessibility updates/fixes. 

This was a quick automated test with some additional manual verification and searching. Happy to go into more detail if anything doesn't make sense. I'm not an expert but have to deal with accessibility regularly with my day job.

**Comment by nigel-bmlt** [comment]
will split these up so we can reopen if i didn't fix them correctly.

**Comment by nigel-bmlt** [comment]
fixed in 0.3.4
